CONTRIBUTING_NEW.md

```diff
@@ -89,6 +92,9 @@ LANGUAGES="javascript ruby" make
 PACKAGES="messages gherkin" make
 ```
 
+Package names are the top level directory names, and language names are the
```

Comment:
you could also add here if you wanted to they are the programming languages i.e. javascript/ruby/java